privacy_evaluator/output/user_output_property_inference_attack.py

```diff
@@ -0,0 +1,14 @@
+from .user_output import UserOutput
+from typing import Dict
+
+
+class UserOutputPropertyInferenceAttack(UserOutput):
+
+    """User Output for Inference Attacks Class"""
+
+    def __init__(self, max_message: str, output: Dict[str, float]):
+        """
+        Initilaizes the Class with values
```

Comment:
I it methods usually do not require documentation apart from Maybe the parameters. Otherwise, their function is clear. But good start with a verb in the 3rd person singular!!

Reply:
You're right. It's updated.